docs/update-bcd.md

```diff
@@ -25,6 +25,14 @@ npm run update-bcd ../local-results
 npm run update-bcd ../mdn-bcd-results/9.1.0-chrome-112.0.0.0-mac-os-10.15.7-79d130f929.json
 ```
 
+Users are advised to restrict the scope of updates, limiting to particular APIs, and mirroring mobile browsers that use the same engines as the desktop browsers.
+This can reduce the number of values that need to be corrected.
+For example, you could update just the WebTransport APIs using desktop browser information, so that mobile browsers are mirrored:
+
+```sh
+npm run update-bcd -- --path=api.WebTransport -b chrome -b firefox -b safari 
+```
+
```

Comment:
I can't say that we can "advise" these steps, as this would only apply for one use case.  In my own use case, running the script with no arguments and sifting through the results is what works best, as my goal is to sync _all_ of the collector's results with BCD.